src/labone/nodetree/node.py

```diff
@@ -141,21 +141,6 @@ def path(self) -> str:
         """LabOne path of the node."""
         return self._info["Node"].lower()
 
-    @property
-    def description(self) -> str:
-        """Description of the node."""
-        return self._info["Description"]
-
-    @property
-    def type(self) -> str:  # noqa: A003
-        """Type of the node."""
-        return self._info["Type"]
-
-    @property
-    def unit(self) -> str:
-        """Unit of the node."""
-        return self._info["Unit"]
-
```

Comment:
issue: I know why you can remove this but its not user friendly. A user of the API does most likely not know that these keys are present in the node info. Therefore I would still hard-code them

Reply:
I could write all viable options in the documentation of the class or the getattr - would that be acceptable?